packages/paypal-commerce-integration/.eslintrc.json

```diff
@@ -0,0 +1,42 @@
+{
+    "extends": ["../../.eslintrc.json"],
+    "ignorePatterns": ["!**/*"],
+    "overrides": [
+        {
+            "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+            "rules": {
+                "@typescript-eslint/naming-convention": "off",
```

Comment:
This seems ok as of now. But we should try and fix whatever we can for now, and maybe create items to fix these items as well.

Reply:
I removed almost all ts-lint naming-convention errors, but we should keep this rule off due to the bigpay or paypal sdk specific variable names. Just don't want to use ts-ignore all over the place